src/test/kotlin/uk/gov/justice/digital/hmpps/manageoffencesapi/resource/OffencesControllerIntTest.kt

```diff
@@ -103,118 +100,4 @@ class OffencesControllerIntTest : IntegrationTestBase() {
         )
     }
   }
-
```

Comment:
this endpoint no longer exists - the full sync to nomis  is no longer directly triggered by an api point and insread runs on a schedule protected by a feature toggle